lib/rack/tracker.rb

```diff
@@ -55,7 +55,11 @@ def html?; @headers['Content-Type'] =~ /html/; end
 
     def inject(env, response)
       @handlers.each(env) do |handler|
-        response.gsub!(%r{</#{handler.position}>}, handler.render + "</#{handler.position}>")
+        if handler.container_position == :closing
+          response.gsub!(%r{</#{handler.container_tag}>}, handler.render + "</#{handler.container_tag}>")
```

Comment:
I mean, yes it's a first draft. But what I totally dislike about this code is the responsibility of the tracker.
It was not optimal before, but now it's even worse. Why do the tracker need to ask the handler about his position, to again decide which "message" to send to it again.... weird. The handler should take care of all that.